base/gitserver/gitserver.StatefulSet.yaml

```diff
@@ -20,6 +20,15 @@ spec:
         group: backend
         type: gitserver
     spec:
+      initContainers:
+      - name: transfer-file-ownership
+        image: sourcegraph/alpine:3.10@sha256:4d05cd5669726fc38823e92320659a6d1ef7879e62268adec5df658a0bacf65c
+        command: ["sh", "-c", "chown -R 100:100 /data/repos"]
```

Comment:
is there a way to skip running this if the perms are fine already? this could be a slow operation on large disks.

Reply:
well observed. i will try a couple of things. maybe we can do something here

Reply:
i did it for gitserver which has big disks